### PR TITLE
feat: added a bookmark icon and a modal to display saved cities

### LINKED ISSUE #13: Bookmark cities
By clicking the bookmark icon when a city has been selected, we can save it and view it on the bookmark modal.

### PR DESCRIPTION
**Pull request by KoichiMin**
- [x] feat : implemented a bookmark icon to offer a option to save the weather data of the selected location 
- [x] feat : added a button to display the weather data of the bookmarked cities by implementing a react modal
- [x] chore : modal and cities displayed are mobile responsive 

Closes #13

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -12205,4 +12205,4 @@ supports-color@^5.3.0, supports-color@^5.5.0:
 yocto-queue@^0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/yocto-queue/-/yocto-queue-0.1.0.tgz#0294eb3dee05028d31ee1a5fa2c556a6aaf10a1b"
-  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
\ No newline at end of file
+  integrity sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==
```
Comment (SySagar) [inline yarn.lock]: Why are the lock files changed?

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_APIKEY=b17a235a27e088c0f6920199baa7f36d
```
Comment (SySagar) [inline .env:1]: The env file should not be commited. Please put it inside .gitignore.

**Comment by KoichiMin** [comment]
@SySagar We finished our footer so that can be merge into the main Repo

**Review by SySagar — APPROVED** [review]

**Comment by SySagar** [comment]
LGTM!

**Pull request merged**